lara-typescript/src/section-authoring/components/section-item-picker.scss

```diff
@@ -131,23 +131,25 @@ input, select {
     padding: 0;
 
     li button {
-      background: transparent;
+      background: transparent !important;
```

Comment:
Maybe we should set a class on the button instead of playing the `!important` whack-a-mole game?